Lombiq.DataTables/Models/DataTableColumnDefinition.cs

```diff
@@ -16,7 +16,7 @@ public class DataTableColumnDefinition
 
     public string this[string key]
     {
-        get => AdditionalSettings.ContainsKey(key) ? AdditionalSettings[key]?.ToString() : null;
+        get => AdditionalSettings.TryGetValue(key, out var settings) ? settings?.ToString() : null;
```

Comment:
The `out` object is guaranteed not to be `null`:
![image](https://github.com/Lombiq/Orchard-Data-Tables/assets/164360/c4d9e9ae-948c-4aec-a404-fb122b81c601)

Thus, we can drop the null check:
```suggestion
        get => AdditionalSettings.TryGetValue(key, out var settings) ? settings.ToString() : null;
```